tests/testCameraMapper.py

```diff
@@ -207,6 +209,31 @@ def testSubMap(self):
                          'ccd = 13\nheight = 400\nimageOrigin = "PARENT"\n'
                          'llcX = 200\nllcY = 100\nwidth = 300\n')
 
+    def testCatalogExtras(self):
+        butler = dafPersist.ButlerFactory(mapper=self.mapper).create()
```

Comment:
Do you really need to do this via a ButlerFactor? Can't you just create a butler directly?